Source/Core/Core/HW/DSPHLE/UCodes/Zelda.h

```diff
@@ -158,7 +158,7 @@ class ZeldaAudioRenderer
   // the Wii, this points to some MRAM location since there is no ARAM to be
   // used. If zero, use the top of ARAM.
```

Comment:
This comment probably should be rewritten, since `m_aram_base_addr = 0` no longer behaves differently (and `SetARAMBaseAddr` would never be called because command 0xE doesn't exist in GameCube uCode). I'm not 100% sure what it should say though.

Reply:
Came up with something, hopefully the wording is clear enough.